product-pricing-service/src/main/java/com/hedvig/contract/type/danishHomeContent/certificates/DanishHomeContentCertificate.kt

```diff
@@ -0,0 +1,18 @@
+package com.hedvig.contract.type.danishHomeContent.certificates
+
+import com.hedvig.contract.type.danishHomeContent.aggregates.DanishHomeContentAgreement
+import com.hedvig.contract.shared.certificates.InsuranceCertificateTemplate
+import com.hedvig.contract.shared.certificates.MemberInfo
+import java.time.LocalDate
+
+class DanishHomeContentCertificate(
+  override val inceptionDate: LocalDate,
+  override val holderMemberId: String,
+  override val agreement: DanishHomeContentAgreement,
+  override val memberInfo: MemberInfo,
+  override val isDraft: Boolean
+) : InsuranceCertificateTemplate() {
+  override fun getInsuranceCertHtmlFile(): String {
+    return "insurance_certificate_den_home_content"
```

Comment:
"den" looks a bit weird to me
```suggestion
    return "insurance_certificate_dk_home_content"
```

Reply:
In norway its nor and in sweden it's see should we change all of them then?